routes/api.php

```diff
@@ -66,10 +62,9 @@
 Route::get('/profile/{address}',[ProfileController::class, 'getProfile']);
 Route::get('/protocols', [DataController::class, 'getProtocols']);
 Route::get('/circles', [CircleController::class, 'getCircles']);
-//// not used for now
-//Route::post('/circles', [CircleController::class, 'createCircle']);
-////
-
+Route::middleware(['verify-sign-only'])->group(function() {
```

Comment:
shouldn't the existing `verify-sign` work for this? because there's no `$request->route('circle_id')`

Reply:
yeah this makes sense